cmd/wasm/functions.go

```diff
@@ -5,13 +5,14 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"reflect"
+	"syscall/js"
+
 	"github.com/speakeasy-api/jsonpath/pkg/jsonpath"
 	"github.com/speakeasy-api/jsonpath/pkg/jsonpath/config"
 	"github.com/speakeasy-api/jsonpath/pkg/jsonpath/token"
 	"github.com/speakeasy-api/jsonpath/pkg/overlay"
 	"gopkg.in/yaml.v3"
-	"reflect"
-	"syscall/js"
```

Comment:
VSCode / go format didn't approve of the format for many of these files. I can drop this commit if that's a problem...